packages/stats/gbstats/gbstats.py

```diff
@@ -165,12 +165,19 @@ def analyze_row(s):
                 stat_a, RegressionAdjustedStatistic
             ):
                 theta = compute_theta(stat_a, stat_b)
-                stat_a.theta = theta
-                stat_b.theta = theta
+                if theta == 0:
+                    # revert to non-RA under the hood if no variance in a time period
+                    stat_a = stat_a.post_statistic
+                    stat_b = stat_b.post_statistic
+                else:
+                    stat_a.theta = theta
+                    stat_b.theta = theta
 
             s[f"v{i}_cr"] = stat_b.unadjusted_mean
             s[f"v{i}_expected"] = (
-                (stat_b.mean / stat_a.mean) - 1 if stat_a.mean > 0 else 0
+                (stat_b.mean - stat_a.mean) / stat_a.unadjusted_mean
+                if stat_a.unadjusted_mean > 0
+                else 0
```

Comment:
This is the only change that is on an actively used path, and unadjusted_mean should always == mean for non-RA metrics.